### DISCUSSION TITLE
Verknüpfung der valuesets

### DISCUSSION THREAD

**Discussion by msteffenDev** [Digitaler-Impfnachweis/certification-apis#84] [Q&A]
gibt es eine Vernüpfungstabelle, aus der hervorgeht, wie die Impstoffe mit den Hersteller und den Impstofftypen verknüpft sind. Ich habe so etwas in den valuesets nicht gefunden.

Vielen Dank!

**Comment by thinkberg** [comment]
Auf der EU eHealth Network Seite gibt es unter Valuesets eine Liste: https://ec.europa.eu/health/ehealth/covid-19_en

**Reply by Ein-Tim** [reply]
@thinkberg 

Could you please let us know wether it is planned to respond to the issues in https://github.com/Digitaler-Impfnachweis/documentation/? 

If not this should be communicated clearly. 

Thank you, have a nice night!
